package.json

```diff
@@ -30,6 +30,6 @@
     "typescript": "4.1.2"
   },
   "peerDependencies": {
-    "@multiversx/sdk-core": ">= 12.1.0"
+    "@multiversx/sdk-core": ">= 12.18.0"
```

Comment:
Referenced the newer (non-breaking) version of sdk-core in order to fix some npm audit issues.